src/thor/dijkstras.cc

```diff
@@ -12,9 +12,9 @@ using namespace valhalla::sif;
 namespace {
 
 // Method to get an operator Id from a map of operator strings vs. Id.
-uint32_t GetOperatorId(const GraphTile* tile,
-                       uint32_t routeid,
-                       std::unordered_map<std::string, uint32_t>& operators) {
+uint32_t GetOperatorIdDS(const GraphTile* tile,
```

Comment:
Also same implementation as in 
https://github.com/valhalla/valhalla/blob/de546d76250402d0f37d06eebe665fe443788a71/src/thor/multimodal.cc#L14

Believe should extract somewhere (also not related to proto stuff, so not to `proto_conversions`)

Reply:
Done